css/search.css

```diff
@@ -0,0 +1,96 @@
+   * {
+  box-sizing: border-box;
+}
+.search {
+  width: 100px;
+  height: 100px;
+  margin: 40px auto 0;
+  background-color: #00FEDE;
+  position: relative;
+  overflow: hidden;
+  transition: all 0.5s ease;
+  border-radius: 70px;
+}
+.search:before {
+  content: '';
+  display: block;
+  width: 3px;
+  height: 100%;
+  position: relative;
+  background-color: #00FEDE;
+  transition: all 0.5s ease;
+}
+.search.open {
+  width: 420px;
+  border-radius: 70px;
+}
+.search.open:before {
+  height: 60px;
+  margin: 20px 0 20px 30px;
+  position: absolute;
+}
+
+.search-box {
+  width: 75%;
+  height: 100%;
+  box-shadow: none;
+  border: none;
+  color: #969191;
+  padding: 20px 100px 20px 45px;
+  font-size: 40px;
+}
+.search-box:focus {
+  outline: none;
+}
+
+.search-button {
+  width: 100px;
+  height: 100px;
+  display: block;
+  position: absolute;
+  right: 0;
+  top: 0;
+  padding: 20px;
+  cursor: pointer;
+}
+
+.search-icon {
+  width: 40px;
+  height: 40px;
+  border-radius: 40px;
+  border: 3px solid #fff;
+  display: block;
+  position: relative;
+  margin-left: 5px;
+  transition: all 0.5s ease;
+}
+.search-icon:before {
+  content: '';
+  width: 3px;
+  height: 15px;
+  position: absolute;
+  right: -2px;
+  top: 30px;
+  display: block;
+  background-color: #fff;
+  transform: rotate(-45deg);
+  transition: all 0.5s ease;
+}
+.search-icon:after {
+  content: '';
+  width: 3px;
+  height: 15px;
+  position: absolute;
+  right: -12px;
+  top: 40px;
+  display: block;
+  background-color: #fff;
+  transform: rotate(-45deg);
+  transition: all 0.5s ease;
+}
+.open .search-icon {
+  margin: 0;
+ 
+  border-radius: 60px;
```

Comment:
Again the border-radius :) with a border-radius of 50% you will not be forced to call it 2 times with diferent values to make the container a perfect circle, just do border-radius : 50%